sensehat-display.c

```diff
@@ -48,7 +48,13 @@ static const u8 gamma_presets[][GAMMA_SIZE] = {
 	},
```

Comment:
The code does not compile after this commit, even though it is fixed in the next one, it is generally preferable to make sure that the code is always in a "working" state after each commit since it helps make the flow of the changes more clear and makes each commit properly atomic. I think that this and the next commit could be combined (I think you can do this with a sqaush commit, but I am not an expert) though what I would probably do is keep it as two commits, just do the one that is currently second first (just move the function, and put a note in the commit explaining that it is in prep for removing the decl) and then in the second commit actually just remove the forward declaration. This way both commits leave the repo in a valid state and each is more atomic (only touches a single thing)